src/jedi_bundle/config/bundles/soca.yaml

```diff
@@ -8,6 +11,7 @@ required_repos:
   - ufo
   - vader
   - fms
+  - icepack
```

Comment:
I don't mind that `icepack` gets build no matter what because it is lightweight. I'm just wondering why is it not an optional repo? Maybe I'm confusing the logic with `jedi_bundle` yaml and Cmake `if found` option.

Reply:
Done